hypertrace-graphql-labels-schema-impl/src/main/java/org/hypertrace/graphql/label/dao/LabelResponseConverter.java

```diff
@@ -60,5 +66,36 @@ private Label convertLabel(org.hypertrace.label.config.service.v1.Label label) {
     String key;
     String color;
     String description;
+    Map<String, List<LabeledEntity>> labeledEntitiesMap;
+
+    @Override
+    public LabeledEntityResultSet labeledEntities(String entityType, int limit) {
+      List<LabeledEntity> labeledEntities =
+          Optional.ofNullable(labeledEntitiesMap.get(entityType)).orElse(Collections.emptyList());
+      List<LabeledEntity> labeledEntitiesWithLimit =
+          labeledEntities.size() > limit ? labeledEntities.subList(0, limit) : labeledEntities;
```

Comment:
limits should be enforced on the request side

Reply:
An entity may hold multiple labels. I was planning to fetch entities along with labels, and reverse map from labels to entities here. Let me know if there is a better approach.

Reply:
I was thinking we'd just create a query for entities filtered by label. Using the label results, you have a list of label IDs which can become an IN filter on a single entity query (per requested type). You're likely right about the limit being enforced on the response side though, now that I think about it. We can either
- Make a request per [label, type] pair and enforce it per request
- Make a request per type, and enforce it in memory when demuxing the result
The second option seems to me like it will be significantly more performant, even if I don't love the idea that we're limiting in memory.

Reply:
I was going for second option as well